### PR TITLE
5.8.0 release notes

### PR DESCRIPTION
**Pull request by id**

### CODE REVIEW

File: en_US/changes/changes-ee-v5.md [hunk 1]
```diff
@@ -1,5 +1,180 @@
...
+
+- [#13463](https://github.com/emqx/emqx/pull/13463) Now, when GCP PubSub Producer action receives a response from PubSub with the HTTP status codes 502 or 503, it'll retry the request until it succeeds or the message TTL is reached.
+
+- [#13493](https://github.com/emqx/emqx/pull/13493) Upgrade RPC library `gen_rpc` to 3.4.0
+
+  The default RPC server socket option in `gen_rpc` 3.4.0 has been changed from `true` to `active-100` which should provide back-pressure towards peer nodes when the RPC server is under heavy load.
+
+- [#13546](https://github.com/emqx/emqx/pull/13546) Added the option to configure the query mode for Pulsar Producer action.
+
+- [#13560](https://github.com/emqx/emqx/pull/13560) Introduced a Thrift driver for the IotDB connector.
```
Comment (hjianbo) [inline en_US/changes/changes-ee-v5.md]:
```suggestion
```

Due to we moved this support to e5.8.1

**Comment by ysfscream** [comment]
Hi all. I've updated the CE 5.8.0 limitations for Dashboard in the breaking changes documentation.

**Comment by Meggielqk** [comment]
I didn't see the Cluster Linking feature.

File: en_US/changes/changes-ee-v5.md [hunk 2]
```diff
@@ -1,5 +1,187 @@
 # Version 5
 
+## 5.8.0
+
+*Release Date: 2024-08-28*
+
+### Enhancements
+
+- [#13126](https://github.com/emqx/emqx/pull/13126) Implement Cluster Linking feature. Cluster Linking is a feature that connects multiple, separate EMQX clusters, facilitating communication between clients on different, often geographically dispersed clusters.
```
Comment (id) [inline en_US/changes/changes-ee-v5.md]: @zmstone not sure which category to put this under

**Comment by id** [comment]
> I didn't see the Cluster Linking feature.

fixed

File: en_US/changes/changes-ce-v5.md [hunk 3]
```diff
@@ -1,5 +1,133 @@
...
+        payload.path.to.value
+    END AS my_value
+  ```
+
+  you can now write a more concise expression: `SELECT coalesce(payload.path.to.value, 0) AS my_value`.
+
+
+#### Operations
+
+- [#13202](https://github.com/emqx/emqx/pull/13202) Introduced the `emqx_cli conf cluster_sync fix` command to address cluster configuration inconsistencies. This command synchronizes the configuration of all nodes with the configuration of the node that has the highest `tnx_id`, ensuring consistency across the cluster.
```
Comment (hjianbo) [inline en_US/changes/changes-ce-v5.md]:
```suggestion
- [#13202](https://github.com/emqx/emqx/pull/13202) Introduced the `emqx ctl conf cluster_sync fix` command to address cluster configuration inconsistencies. This command synchronizes the configuration of all nodes with the configuration of the node that has the highest `tnx_id`, ensuring consistency across the cluster.

```

File: en_US/changes/changes-ee-v5.md [hunk 4]
```diff
@@ -1,5 +1,183 @@
...
+
+- [#13463](https://github.com/emqx/emqx/pull/13463) Enhanced the GCP PubSub Producer action to automatically retry requests when receiving HTTP status codes 502 (Bad Gateway) or 503 (Service Unavailable) from PubSub. The retries will continue until the request is successful or the message's Time-To-Live (TTL) is reached.
+
+- [#13546](https://github.com/emqx/emqx/pull/13546) Added a configurable option for the query mode in the Pulsar Producer action, allowing users to customize how data is queried before it is sent to the Pulsar service.
+
+- [#13650](https://github.com/emqx/emqx/pull/13650) EMQX now supports data integration with DataLayers.
+
+#### Operations
+
+- [#13202](https://github.com/emqx/emqx/pull/13202) Introduced the `emqx_cli conf cluster_sync fix` command to address cluster configuration inconsistencies. This command synchronizes the configuration of all nodes with the configuration of the node that has the highest `tnx_id`, ensuring consistency across the cluster.
```
Comment (hjianbo) [inline en_US/changes/changes-ee-v5.md]:
```suggestion
- [#13202](https://github.com/emqx/emqx/pull/13202) Introduced the `emqx ctl conf cluster_sync fix` command to address cluster configuration inconsistencies. This command synchronizes the configuration of all nodes with the configuration of the node that has the highest `tnx_id`, ensuring consistency across the cluster.
```

File: zh_CN/changes/changes-ce-v5.md [hunk 5]
```diff
@@ -1,5 +1,117 @@
...
+      ELSE
+        payload.path.to.value
+    END AS my_value
+  ```
+
+  现在可以使用更简洁的表达式：`SELECT coalesce(payload.path.to.value, 0) AS my_value`。
+
+#### 运维
+
+- [#13202](https://github.com/emqx/emqx/pull/13202) 引入了 `emqx_cli conf cluster_sync fix` 命令，用于解决集群配置不一致的问题。该命令将所有节点的配置与具有最高 `tnx_id` 的节点的配置进行同步，确保集群配置的一致性。
```
Comment (hjianbo) [inline zh_CN/changes/changes-ce-v5.md]:
```suggestion
- [#13202](https://github.com/emqx/emqx/pull/13202) 引入了 `emqx ctl conf cluster_sync fix` 命令，用于解决集群配置不一致的问题。该命令将所有节点的配置与具有最高 `tnx_id` 的节点的配置进行同步，确保集群配置的一致性。

```

File: zh_CN/changes/changes-ee-v5.md [hunk 6]
```diff
@@ -1,5 +1,168 @@
...
+
+- [#13463](https://github.com/emqx/emqx/pull/13463) 增强了 GCP PubSub 生产者动作，当从 PubSub 接收到 HTTP 状态代码 502（Bad Gateway）或 503（Service Unavailable）时，自动重试请求。重试将持续进行，直到请求成功或消息的生存时间（TTL）到达。
+
+- [#13546](https://github.com/emqx/emqx/pull/13546) 为 Pulsar 生产者动作添加了一个可配置的查询模式选项，允许用户在数据发送到 Pulsar 服务之前自定义数据查询方式。
+
+- [#13650](https://github.com/emqx/emqx/pull/13650) EMQX 支持与 DataLayers 的数据集成。
+
+#### 运维
+
+- [#13202](https://github.com/emqx/emqx/pull/13202) 引入了 `emqx_cli conf cluster_sync fix` 命令，用于解决集群配置不一致的问题。该命令将所有节点的配置与具有最高 `tnx_id` 的节点的配置进行同步，确保集群配置的一致性。
```
Comment (hjianbo) [inline zh_CN/changes/changes-ee-v5.md]:
```suggestion
- [#13202](https://github.com/emqx/emqx/pull/13202) 引入了 `emqx ctl conf cluster_sync fix` 命令，用于解决集群配置不一致的问题。该命令将所有节点的配置与具有最高 `tnx_id` 的节点的配置进行同步，确保集群配置的一致性。
```

**Review by yanzhiemq — APPROVED** [review]

**Pull request merged**